examples/SuperResolution/voxel-ShapeNet/train.py

```diff
@@ -0,0 +1,212 @@
+import argparse
+import json
+import numpy as np
+import os
+import torch
+from torch.autograd import Variable
+import torch.optim as optim
+from torch.utils.data import DataLoader
+import sys
+from tqdm import tqdm
+
+from utils import up_sample
+from architectures import EncoderDecoder_32_128, EncoderDecoderForNLL_32_128
+
+import kaolin as kal
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--shapenet-root', type=str, required=True, help='Root directory of the ShapeNet dataset.')
+parser.add_argument('--cache_dir', type=str, default='cache', help='Root directory of the ShapeNet dataset.')
+parser.add_argument('--loss-type', type=str, choices=['MSE', 'NLLL'], default='MSE', help='Specify the loss type to use.')
+parser.add_argument('--device', type=str, default='cuda', help='Device to use')
+parser.add_argument('--categories', type=str, nargs='+', default=['chair'], help='list of object classes to use.')
+parser.add_argument('--epochs', type=int, default=30, help='Number of train epochs.')
+parser.add_argument('--batchsize', type=int, default=16, help='Batch size.')
+parser.add_argument('-lr', '--learning_rate', type=float, default=1e-3, help='Learning rate.')
```

Comment:
like comment above: `--lr`

Reply:
You're totally right. Fixed one but not the other.

Reply:
Actually it's better this way, didn't see that there were the option '--learning_rate'